composer.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6",
```

Comment:
Thanks Markus, `phpecc` needs PHP 5.6 and I wasn't sure at the time if composer handled dependency requirements. By the way, I clearly indicate in the Readme that 5.6 is needed, so if Composer handles them, we can require a previous version.